### ISSUE TITLE
Not achieving good result on MegaFace

### ISSUE THREAD

**Issue by Paralysis** [wy1iu/sphereface#8]
First of all, thank you for providing the Sphereface training code for us. 

I have retrained your model following your instruction. However, I only got 67.35% on Rank-1 Identification Accuracy on Megaface testing, while you got 72.73%. We used your code and trained from scratch. 

Training with Softmax + CenterLoss rendered the correct result as reported in your paper, i.e. ~65%. 

I believe this is the same protocol you used for Megaface Testing, right? If not, could you kindly point out the difference? Thank you very much!

**Comment by ydwen** [comment]
Hi Paralysis.
The released model is not what we used for Megaface.

**Comment by Paralysis** [comment]
Hi ydwen,
I think you just changed the protocol to 20 layer architecture.
But I mean, I trained on the 28 layer architecture you previously uploaded. The one from the Center Loss paper. Is this the same model you used for MegaFace test? 

Thanks for your quick reply.

**Comment by wy1iu** [comment]
Neither 28 layer nor 20 layer architecture is used for Megaface.

**Comment by bkj** [comment]
@ydwen @wy1iu Are you able to release the pretrained model and/or architecture used for Megaface, so that we can try to reproduce the results?

**Comment by Paralysis** [comment]
Same question as @bkj. I would appreciate it if you share with the community the architecture.

**Comment by kalyo-zjl** [comment]
Hi @Paralysis,

Could you please kindly tell me how to get the Rank-1 Identification Accuracy on Megaface. I have trained a model of ~99.3 on LFW dataset. But the result on Megaface is very bad, the result under distractors 1000000 is only 0.4. I just use run_experiment.py in Megaface devkit to get the result. Can you give me some advice about how you get your Accuracy number of 67.35%?

Thanks in advance.
